library/src/com/squareup/timessquare/CalendarPickerView.java

```diff
@@ -814,6 +886,9 @@ private static boolean sameMonth(Calendar cal, MonthDescriptor month) {
   }
 
   private boolean isDateSelectable(Date date) {
+    if (isCheckDayInSelectableCals && !containsDate(selectableCals, date)) {
```

Comment:
That sounds good to me. I wasn't thinking too hard when I reviewed this. Just passing over with normal style guide stuff.